feat(kinputswitch): reskin component [KHCP-8986] (#1871)
Kong/kongponents: docs/components/input-switch.md

# Input Switch

> KInputSwitch, similarly to checkbox, is used to toggle values on and off.

Reviewer: ```suggestion
KInputSwitch, similar to a checkbox, is used to toggle a value on and off.
```

<KInputSwitch label="Toggle" v-model="vModel0" />

```html
<KInputSwitch label="Toggle" v-model="switchValue" />
```

## Props

### label

Will place label text to the right of the switch. Can also be [slotted](#slots).

> <KInputSwitch v-model="vModel3" label="Label" />

Reviewer: For the text above, can you change to this:

```
A `string` of text to display as a label after the input switch. The label can also be [slotted](#slots).
```

```html
<KInputSwitch v-model="switchValue" label="Label" />
```

> ### size

Reviewer: Can you move the `size` prop to right above the `disabledTooltipText` prop?

KInputSwitch comes in two sizes: `small` (default) and `large`.

<div class="vertical-spacing">
  <KInputSwitch size="small" label="Small" v-model="vModel1" />
  <KInputSwitch size="large" label="Large" v-model="vModel2" />
</div>

```html
<KInputSwitch size="small" label="Small" v-model="switchValue1" />
<KInputSwitch size="large" label="Large" v-model="switchValue2" />
```

### labelBefore

> Use this prop if you want the label to render on the left of the toggle box.

Reviewer: A `boolean` to place the label before the input switch. Defaults to `false`.

<KInputSwitch label-before label="Label before" v-model="vModel4" />

```html
<KInputSwitch label-before label="Label before" v-model="switchValue" />
```

### v-model

> Value to bind KSwitchInput value to.

Reviewer: typo
```suggestion
Value to bind to the KInputSwitch.
```

Reviewer: Should we move `v-model` to the top since it's likely the most used?

<KInputSwitch v-model="vModel5" :label="vModel5 ? 'Toggled on' : 'Toggled off'" />

```html
<KInputSwitch v-model="switchValue" :label="switchValue ? 'Toggled on' : 'Toggled off'" />
```

### disabledTooltipText

Use this prop to display tooltip text when KInputSwitch is disabled.

<KInputSwitch disabled disabled-tooltip-text="Scale down this cluster first to enable editing" v-model="vModel6" />

```html
<KInputSwitch disabled disabled-tooltip-text="Scale down this cluster first to enable editing" v-model="switchValue" />
```

### HTML Attributes

#### disabled

> You can add `disabled` attribute to disallow interactivity.

Reviewer: ```suggestion
You can add the `disabled` attribute to disallow interactivity.
```
